src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__information {
+    margin-top: 40px;
+  }
+
+  &__title {
+    color: $primary-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-stars {
+      display: flex;
+      flex-flow: row wrap;
+    }
+
+    &-total {
+      color: $primary-color;
+      text-align: right;
+    }
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__star--active {
+    background-image: url("../images/star-active.svg");
+    background-repeat: no-repeat;
+    background-position: 50%;
+  }
+
+  &__cost {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__amount {
+    text-align: right;
```

Comment:
You already use `flex` with `space between` for parent element. So this property is not necessary. Remove it